javascript/signFunction.js

```diff
@@ -0,0 +1,59 @@
+const inputFunction = {
+  // 이메일 입력란에 focus 일 때 이벤트 핸들러
+  emailFocus: (e) => {
+    e.target.classList.remove('input-error')
+    e.target.parentElement.lastElementChild.textContent = ''
+  },
+
+  // 이메일 입력란에서 blur 일 때 이벤트 핸들러
+  emailBlur: (e, emailForm, accountInfo, inputEmail, signupValidation = false) => {
+    const matchingAccountEmail = accountInfo.find((item) => item.email === inputEmail?.value)
```

Comment:
해당 변수는 accountInfo 에서 email input 값이랑 일치하는 데이터를 저장한거니 이름이 이를 나타내면 좋겠습니다.

```suggestion
    // 1. 어떤 변수인지 알 수 있도록 이름 변경 
    const matchedAccount = accountInfo.find((item) => item.email === inputEmail?.value)
    // 2. 일치하는 계정에서 email만 추출
    const matchedEmail = accountInfo.find((item) => item.email === inputEmail?.value)?.email;
```

Reply:
네 알겠습니다!! ㅎㅎ